flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/DeleteNodeHandler.java

```diff
@@ -83,13 +86,24 @@ public DeleteNodeHandler(JsonElement nodeToDelete, Path filePath) {
 
     @Deprecated
     public JsonElement getTextEditsToDeletedNode(Document document, Project project) {
+        if ((nodeToDelete.codedata() != null && nodeToDelete.codedata().node() == NodeKind.ERROR_HANDLER)) {
+            return handleErrorHandlerDeletion(nodeToDelete.codedata().lineRange(), filePath, document, project);
+        }
+
```

Comment:
@pasindufernando1 do we need this check in here as we have the same check in the overloaded method we are using? (L102-L104)